spec/core/ics-023-vector-commitments/README.md

```diff
@@ -38,6 +38,9 @@ This document only defines desired properties, not a concrete implementation —
 
 ## Technical Specification
 
+Below we define a behaviour and an overview of datatypes. For data type definition look at [confio/ics23](https://github.com/confio/ics23/blob/master/go/ics23.go) repository.
```

Comment:
Please link to the actual protobuf definitions, which are the data spec, not the Go code, which is derived from them (as is the Rust and TS code)

https://github.com/confio/ics23/blob/master/proofs.proto

Otherwise, glad to see this link

Reply:
done